frontend/javascripts/oxalis/view/action-bar/tracing_actions_view.js

```diff
@@ -397,16 +397,23 @@ class TracingActionsView extends React.PureComponent<Props, State> {
           ),
         ]
       : [
-          <ButtonComponent key="read-only-button" type="danger" disabled>
+          <ButtonComponent
+            key="read-only-button"
+            type="danger"
+            disabled
+            style={{ backgroundColor: "var(--ant-warning-dark-5)" }}
```

Comment:
I changed the bg color of the read-only button from red to orange (as it's been before the antd update).